chainerx_cc/chainerx/cuda/numeric.cuh

```diff
@@ -62,8 +74,6 @@ CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Arcsin, std::asin)
 CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Arccos, std::acos)
 CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Arctan, std::atan)
 CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Exp, std::exp)
-CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Expm1, std::expm1)
-CHAINERX_DEFINE_CUDA_FLOAT16_FALLBACK_UNARY(Exp2, std::exp2)
```

Comment:
Doesn't this macro work?
As `std::expm1` and `std::exp2` have `float` overloads, we don't have to use special versions like `std::expm1f`.

Reply:
I made this change because I was getting the following error earlier for `float` type inputs.
```
home/akshay/chainer/chainerx_cc/chainerx/cuda/numeric.cuh(40): error: calling a constexpr __host__ function("expm1") from a __device__ function("Expm1") is not allowed. The experimental flag '--expt-relaxed-constexpr' can be used to allow this.

/home/akshay/chainer/chainerx_cc/chainerx/cuda/numeric.cuh(49): error: calling a constexpr __host__ function("exp2") from a __device__ function("Exp2") is not allowed. The experimental flag '--expt-relaxed-constexpr' can be used to allow this.
```

Reply:
I see. Thank you for clarification!